### ISSUE TITLE
Copy buttons for commands are missed on mobile

### ISSUE THREAD

**Issue by alex-key** [npmx-dev/npmx.dev#1771]
### Describe the bug

On Package page each command has a "Copy" button next to it. It is visible on hover state for the desktop version. But it is missed on mobile devices since there is no hover state.

**Steps to reproduce:**

- Use mobile device or emulator in your browser
- Go to https://npmx.dev/package/nuxt#get-started
- In section "GET STARTED" check no button is visible to copy commands
- Still copied if you click on command, but it's not intuitive

### Additional context

<img width="583" height="1160" alt="Image" src="https://github.com/user-attachments/assets/7f0a79f1-6b89-49ca-9338-cece23ab5286" />

<img width="1243" height="724" alt="Image" src="https://github.com/user-attachments/assets/1b1a1e00-ccbc-4f06-b390-2ff0f43e37a5" />

**Comment by ackzell** [comment]
Hi @alex-key, 

What is the expected behavior that is an acceptable fix for this?

While attempting to reproduce I noticed is that I can "tap" on the command on my mobile and I do get the copy button to appear. From there, I do get the command on the clipboard if I tap on the `copy` button.

Is the idea to always show the button for mobile?

**Comment by alex-key** [comment]
From my point on touchscreen device it should be always visible. 
cc: @alexdln

**Comment by alexdln** [comment]
Another problem is that we don't have much space on mobile. But perhaps we could implement our standard behavior here as well: icon + text on desktop, and just the icon on mobile (_and always show it_)

**Comment by RevanthPrk** [comment]
I have fixed this issue.
On mobile devices (xs, sm, md screen sizes), it will now be displayed in this way.

<img width="282" height="497" alt="Image" src="https://github.com/user-attachments/assets/321f30ec-1dac-4d08-8c4d-fcd20a88e1e3" />

In Desktop and laptop screens, it will remain as is

<img width="1176" height="572" alt="Image" src="https://github.com/user-attachments/assets/12adba6c-a7dc-4109-9059-a30c6919ded2" />

If this looks fine, then I will raise a PR @alex-key

**Comment by alex-key** [comment]
I think it's ok. Icon looks a but big to me, but let's keep it as it should be compatible with a11y requirements.
If there are any better options - we'll see during review

**Comment by RevanthPrk** [comment]
Sure, I will raise a PR now
